Makefile

```diff
@@ -64,9 +64,9 @@ define NEWLINE
 
 endef
 
-# 30 minutes is the upper bound defined for all tests, the longer running ones at the time of writing are XDC tests. The
-# 30 minute timeout is also defined in run-tests.yml
-TEST_TIMEOUT ?= 30m
+# 40 minutes is the upper bound defined for all tests, the longer running ones at the time of writing are XDC tests. The
+# 40 minute timeout is also defined in run-tests.yml
+TEST_TIMEOUT ?= 40m
```

Comment:
there are two spots that need to be updated now, and the values need to be > this value. see [`2ccbe18` (#6595)](https://github.com/temporalio/temporal/pull/6595/commits/2ccbe1881bf143ddda7f0a27491a823d268481a3)

Reply:
(btw 6595 makes xdc tests run in < 10 minutes. I'll reduce this after that goes in.)

Reply:
oh wow!